centraldogma/src/main/java/com/linecorp/decaton/centraldogma/CentralDogmaPropertySupplier.java

```diff
@@ -103,6 +117,21 @@ public CentralDogmaPropertySupplier(CentralDogmaRepository centralDogmaRepositor
         } catch (TimeoutException e) {
             throw new RuntimeException(e);
         }
+
+        rootWatcher.watch(node -> {
+            node.fields().forEachRemaining(entry -> {
```

Comment:
In contrast to current approach, new code sets the watch against entire file and iterates through all properties in the file.
Do you have any thoughts if it will cause some downside comparing to current approach?

Reply:
oops, we might want to avoid using `node.fields()` in this change, and instead explicitly check properties based on `ProcessorProperties#PROPERTY_DEFINITIONS`.
There’s no benefit in processing keys that Decaton doesn’t care about. Let me modify this point, though it's not main part for your question...

Reply:
Thank you for the detailed explanation including the implementation of CD!

I see, then the strategy looks good.